aiohttp/web_response.py

```diff
@@ -323,7 +329,9 @@ async def _do_start_compression(self, coding: ContentCoding) -> None:
         if coding != ContentCoding.identity:
             assert self._payload_writer is not None
             self._headers[hdrs.CONTENT_ENCODING] = coding.value
-            self._payload_writer.enable_compression(coding.value)
+            self._payload_writer.enable_compression(
+                coding.value, self._compression_strategy
```

Comment:
As before, this is going to cause a type error, which we'll see once the CI is running again.

Maybe @asvetlov can answer the best way to proceed. Will need 1 of:

- Change `self._payload_writer` to `StreamWriter`.
- Add this parameter to `AbstractStreamWriter`.
- Use an `isinstance(StreamWriter, ...)` check before using the parameter.

Reply:
@webknjaz What do you think about it?

Reply:
Added `if isinstance(self._payload_writer, StreamWriter)` condition